src/util/address.rs

```diff
@@ -56,6 +58,10 @@ pub enum Error {
     EmptyBech32Payload,
     /// Script version must be 0 to 16 inclusive
     InvalidWitnessVersion(u8),
+    /// Unable to parse witness version from string or opcode value
+    UnparsableWitnessVersion(ParseIntError),
```

Comment:
Docs: is this really returned on opcode parsing failure?

Reply:
Nope, only from string. Fixing